layout/_layout.swig

```diff
@@ -86,9 +86,9 @@
   {% include '_third-party/analytics/lean-analytics.swig' %}
   {% include '_third-party/analytics/firestore.swig' %}
   {% include '_third-party/seo/baidu-push.swig' %}
+  {% include '_third-party/math/index.swig' %}
```

Comment:
Put things into math folder, in case of there are more math engines in the future.

Test complete, and works well.